src/lib/components/power-bar/PowerBar.svelte

```
@@ -112,8 +113,9 @@
 		window.location.assign(href);
 	}
 
-	function goToClass(classId: string) {
-		goToLink(`/classes?classId=${classId}`);
+	async function goToClass(classId: string) {
+		isOpen = false;
+		await goto('/classes', { state: { classId } });
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)